Makefile

```
@@ -31,6 +31,7 @@ SRCS	=	$(SRCD)/main.cpp
 SRCS	+=	$(SRCD)/webserver.cpp
 SRCS	+=	$(SRCD)/arguments/check_argc_number.cpp
 SRCS	+=	$(SRCD)/http/Request.cpp
+SRCS	+=	$(SRCD)/std98/std98.cpp
```

Add source file in the Makefile
```suggestion
SRCS	+=	$(SRCD)/http/Request.cpp
SRCS	+=	$(SRCD)/http/Response.cpp
SRCS	+=	$(SRCD)/http/StatusCodes.cpp
SRCS	+=	$(SRCD)/std98/std98.cpp
```